instance/serializers/instance.py

```diff
@@ -71,23 +71,23 @@ class Meta:
     def to_representation(self, obj):
         output = super().to_representation(obj)
         # Convert the state values from objects to strings:
+        if output['server_status'] is None:
+            output['server_status'] = 'empty'  # 'empty' for backwards compatibility
```

Comment:
@bradenmacdonald I checked the client-side code and it turns out `server_status` wasn't being used in a meaningful way, so I completely removed it from the instance serializer.